h2/src/tools/org/h2/build/Build.java

```diff
@@ -1124,4 +1137,23 @@ protected String getLocalMavenDir() {
         return local;
     }
 
+    private String addNashornJavaScriptEngineIfNecessary(String cp) {
+        if (requiresNashornJavaScriptEngine()) {
+            return cp +
+                    File.pathSeparator + "ext/nashorn-core-" + NASHORN_VERSION + ".jar" +
+                    File.pathSeparator + "ext/asm-commons-" + ASM_VERSION + ".jar" +
+                    File.pathSeparator + "ext/asm-tree-" + ASM_VERSION + ".jar" +
+                    File.pathSeparator + "ext/asm-util-" + ASM_VERSION + ".jar";
+        }
+        return cp;
+    }
+
+    private boolean requiresNashornJavaScriptEngine() {
+        String version = System.getProperty("java.specification.version");
+        if (version.startsWith("1.")) {
+            version = version.substring(2);
+        }
+        return Integer.parseInt(version) >= 15; // Nashorn was removed in Java 15
```

Comment:
You can use `getJavaVersion()` here.

Reply:
Thanks, I didn't see that! Done in 687cc0f7a09bbe614acedcf29ca6f0f5738e92aa.